interview-questions.md

```diff
@@ -3,44 +3,45 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. In a model called Animal that has_many Sightings, what is the name of the foreign key? Would the foreign key be part of the Animal model or the Sightings model?
 
-  Your answer:
+  Your answer: animal_id would be the forign key, and would be in the sighting model as the sightings model belong to a specific animal. 
 
-  Researched answer:
+  Researched answer: A foreign key is a column or group of columns in a relational database table that provides a link between data in two tables.
 
 
 
 2. Which routes must always be passed params and why?
 
-  Your answer:
+  Your answer: creating routes, they must always be passed in params because if there were no params, then the data base could be full of errors, making the data base useless.
```

Comment:
here we are looking for "show, update and delete must be passed params. because these methods need to know which id to be manipulated."